src/code42cli/cmds/users.py

```diff
@@ -182,6 +203,29 @@ def handle_row(**row):
     formatter.echo_formatted_list(result_rows)
 
 
+@bulk.command(name="move")
+@read_csv_arg(headers=_bulk_user_move_headers)
+@format_option
+@sdk_options()
+def bulk_move(state, csv_rows, format):
+    """Move the list of users from the provided CSV."""
```

Comment:
In the help text here, we should say `Change the organization of ... ` to further explain what moving means

Reply:
Updated